vignettes/r2dii-analysis.Rmd

```diff
@@ -25,18 +21,16 @@ library(r2dii.match)
 library(r2dii.analysis)
 ```
 
-We also recommend the following two `tidyverse` packages for data manipulation 
-and plotting:
+We also recommend the following two `tidyverse` packages to manipulate and plot data:
 
 ``` {r tidyverse libraries}
-library(dplyr)
+library(dplyr, warn.conflicts = FALSE)
```

Comment:
Avoid a bunch of messages that are not important to this article -- i.e. keep the reader focused on what matters most.